<?xml version="1.0"?>
<launch>
    <!-- pose config file -->
    <arg name="pose_config" default="$(find sence_controllers)/config/static.pose.yaml" />

    <!-- the static pose server -->
    <node name="static_pose_server" pkg="sence_controllers" type="static_pose_server.py" clear_params="true" output="screen" >
        <rosparam file="$(arg pose_config)" command="load" />
    </node>

</launch>
